k9mail/src/main/java/com/fsck/k9/mailstore/LocalFolder.java

```diff
@@ -1341,7 +1342,7 @@ protected void saveMessage(SQLiteDatabase db, Message message, boolean copy, Map
             cv.put("subject", message.getSubject());
             cv.put("sender_list", Address.pack(message.getFrom()));
             cv.put("date", message.getSentDate() == null
-                    ? System.currentTimeMillis() : message.getSentDate().getTime());
+                    ? SystemClock.elapsedRealtime() : message.getSentDate().getTime());
```

Comment:
This is wrong. This will set the date of the email to the amount of time since the phone rebooted.